sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessagesView.java

```diff
@@ -81,7 +84,7 @@ public LabMessagesView() {
 		addHeaderComponent(samplesViewSwitcher);
 
 		addHeaderComponent(ButtonHelper.createIconButton(Captions.labMessageFetch, VaadinIcons.REFRESH, e -> {
-			fetchLabmessages();
+			checkForConcurrentFetch();
```

Comment:
The method name seems a little misleading to me. checkForConcurrentFetch(...) does not just check, it also initiates the fetch. That's why I propose to either add some comment clarifying that or remame thie method to something like initiateLabMessageFetch(...)